easybuild/easyblocks/i/impi.py

```diff
@@ -58,6 +58,12 @@ def extra_options():
         }
         return IntelBase.extra_options(extra_vars)
 
+    def prepare_step(self):
+        if LooseVersion(self.version) >= LooseVersion('2017.2.174'):
+            super(EB_impi, self).prepare_step(requires_runtime_license=False)
```

Comment:
I think we can leave the option to switch off checking the licence there for people who respect this license agreement but they should need to explicitly add this to their easyconfig (and since we know the restrictions, we should perhaps warn them if they use this option).